### PR TITLE
[14.0] [MIG] partner contact birthplace

### PR DESCRIPTION
**Pull request by primes2h**

### CODE REVIEW

**Review by marcelofrare — APPROVED** [review]
LGTM
Thank you!

**Review by OpenCode — APPROVED** [review]

File: partner_contact_birthplace/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner contact birthplace",
+    "summary": "This module allows to define a birthplace for partners.",
+    "version": "14.0.1.0.0",
+    "category": "Customer Relationship Management",
+    "website": "https://github.com/OCA/partner-contact" "partner_contact_birthplace",
```
Comment (newtratip) [inline partner_contact_birthplace/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/partner-contact",
```

Comment (primes2h) [reply partner_contact_birthplace/__manifest__.py]: @newtratip 
Done, thank you!

**Review by flotho — APPROVED** [review]
Code review
Thanks
LGTM

**Review by flotho — COMMENTED** [review]
code review, LGTM

**Pull request merged**